misc/vm_configs/boards/whl-ipc-i5/board_info.h

```diff
@@ -8,7 +8,7 @@
 #define BOARD_INFO_H
 
 #define MAX_PCPU_NUM			4U
-#define MAX_PLATFORM_CLOS_NUM		0U
+#define HV_SUPPORTED_MAX_CLOS		0U
```

Comment:
Can we add a comment for the MACRO HV_SUPPORTED_MAX_CLOS	to explain what it mean?

Reply:
Added comments for the 3 new defines, will post the updated patch soon. The comments are added to file misc_cfg.h, as
in subsequent commits, these 3 new defines are moved to file misc_cfg.h instead.